lms/envs/common.py

```diff
@@ -1614,6 +1614,9 @@ def _make_mako_template_dirs(settings):
 GOOGLE_ANALYTICS_TRACKING_ID = None
 GOOGLE_ANALYTICS_4_ID = None
 
+######################## GOOGLE TAG MANAGER ###########################
+GOOGLE_TAG_MANAGER_ID = None
```

Comment:
If this merges, it will need a setting annotation. See XBLOCK_MIXINS below for an example.

Reply:
added annotation:
<img width="529" alt="Screenshot 2025-06-06 at 14 47 46" src="https://github.com/user-attachments/assets/efc1bdf3-e479-48b0-9f72-3190d4c3d178" />